OBSDOCS-677: update CMO config map ref for OCP 4.15 (#70762)
openshift/openshift-docs: monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

=== Description

[IMPORTANT]
====

> These settings are deprecated and are planned to be removed in a future {product-title} version. 

Reviewer: ```suggestion
The setting is deprecated and is planned to be removed in a future {product-title} version. 
```

Author: changed

> In the current version, these settings still exist but have no effect.

Reviewer: ```suggestion
In the current version, this setting still exists but have no effect.
```

Author: changed

|===

== MetricsServerConfig

=== Description

> The `MetricsServerConfig` resource defines settings for the Metrics Server component. Note that this setting only applies when the `MetricsServer` feature gate is enabled.

Reviewer: ```suggestion
The `MetricsServerConfig` resource defines settings for the Metrics Server component. Note that this setting only applies when the `TechPreviewNoUpgrade` feature gate is enabled.
```

Author: Fixed
